nautobot/extras/tests/test_api.py

```diff
@@ -1360,8 +1360,7 @@ class JobTest(
         "time_limit": 650,
         "has_sensitive_variables": False,
         "has_sensitive_variables_override": True,
-        "task_queues": ["default", "priority"],
-        "task_queues_override": True,
+        "job_queues": [],
```

Comment:
Better test would be to have specific queue PKs defined in `setupTestData`.

Reply:
It seems like we can't write directly to `ManyRelatedField` because `read_only` is set to `True`. I looked through the codebase and found no example of updating a many-to-many field from the Rest API.

Reply:
Ah good point. Are we going to need a separate model and API endpoint for `job-to-job-queues` M2M in that case? :-(

Reply:
Sorry if I am misunderstanding, but should JobQueueAssignment do the job in this case?

Reply:
Yeah, I'd forgotten/not seen that you have such a model already. It may just need a REST viewset/url added in that case.

Reply:
Ahhh, yeah those are already added from https://github.com/nautobot/nautobot/pull/6103. One of the cons of breaking down bigger stories.